### DISCUSSION TITLE
The PR source branch in the fork is changed, but PR does not update

### DISCUSSION THREAD

**Discussion by IronLu233** [community/community#38629] [Pull Requests]
https://github.com/ckb-js/lumos/pull/453

I have a PR in my contributing project. from my fork branch to main repo.

Yesterday I update the source branch with commit https://github.com/IronLu233/lumos/commit/a070746eb60f3c12786d9f1f7938a03c7d7b2164

But the PR seems does not react on it.
- github actions do not run
- file changes tab does not update
- no branch update info on PR page

**Comment by ossanna16** [comment]
Hi there @IronLu233 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by IronLu233** [comment]
That becomes correct in this afternoon I seen it. But I confirm it does not update in yesterday
